DEC/src/roe/RoeUtility.jl

```diff
@@ -0,0 +1,222 @@
+# """
```

Comment:
Maybe this should just be called "roe.jl" and be included into module.jl with no surrounding module?